be/app/Http/Controllers/AuthController.php

```diff
@@ -32,12 +32,11 @@ public function register(Request $request)
         $token = $user->createToken('permission')->plainTextToken;
 
         return response([
-            'status' => 200,
             'user' => $user,
             'verified_email' => $user->email_verified_at,
             'token' => $token,
             'Message' => 'Registered Succesfully',
-        ], 201);
+        ]);
```

Comment:
- actually this could belongs to `201` not `200`. (i put the description in my previous comment :man: )
```
        return response([
            'user' => $user,
            'verified_email' => $user->email_verified_at,
            'token' => $token,
            'Message' => 'Registered Succesfully',
        ], 201);
```